src/Uno.UI/UI/Xaml/Style/Generic/Generic.xaml

```diff
@@ -7757,7 +7757,6 @@
 					<Grid Margin="{TemplateBinding Padding}" Background="{TemplateBinding Background}">
 						<ContentPresenter x:Name="Text"
                             Content="{TemplateBinding Content}"
```

Comment:
`Content` template binding still required?

Reply:
No, but It seems to also be present in the WinUI templates and the PR that originally added this change only added ContentTemplate (Content was already present).